file-ops.h

```diff
@@ -48,6 +54,18 @@ inline bool read_file(const std::string& filename, std::string& fileContents)
     file.close();
 
     return success;
+#else
+    emscripten_fetch_attr_t attr;
+    emscripten_fetch_attr_init(&attr);
+    strcpy(attr.requestMethod, "GET");
+    attr.attributes = EMSCRIPTEN_FETCH_ATTR_LOAD_TO_MEMORY | EMSCRIPTEN_FETCH_SYNCHRONOUS;
+    emscripten_fetch_t *fetch = emscripten_fetch(&attr, filename.c_str());
+    if (fetch->status == 200) {
+        *fileContents = fetch->data;
+        return true;
+    } else
+        return false;
```

Comment:
This looks interesting but I'm not sure I like adding this to `read_file` 😝. It might be worth adding this to a new file (`bgfx-enscripten/bgfx-enscripten-utils.hpp`?) that does just the added functionality. We then call that function (with the `#ifdef BX_PLATFORM_EMSCRIPTEN` if required) in `main.cpp` instead of `read_file`. What do you think?